1. Clean All

phaeton03 opened the pull request:
Домашнее задание №1 на проверку

Commit:
1. Clean All

kataus commented on .idea/workspace.xml:
```diff
@@ -4,7 +4,20 @@
     <option name="autoReloadType" value="SELECTIVE" />
   </component>
   <component name="ChangeListManager">
-    <list default="true" id="c158df4a-adb7-407f-af0d-9c6a63a30883" name="Default Changelist" comment="1. Первая версия" />
```
Это ваши личные локальные настройки IDE. Не надо это в общий репозиторий помещать

Commit:
1

Commit:
homework_3
1. домашнее задание №3

Commit:
homework_7
1. добавление использукемых сущностей

Commit:
Merge homework_2

phaeton03 committed:
Merge pull request #4 from phaeton03/homework_3

homework_3

Commit:
homework_7
1. добавление тестов и настройка базы через LiquiBase

phaeton03 committed:
Merge pull request #5 from phaeton03/homework_7

Homework 7

phaeton03 committed:
Homework 9 (#6)

* homework_9
1. Реализация задания №7 через EntityManager

* homework_9
1. Реализация задания №7 через EntityManager

* homework_9
1. Add pom file

* homework_9
1. Add ShellCommands.class

* homework_9
1. Correction #1

* homework_9
1) Исправлений замечаний из ПР

Co-authored-by: u19223669 <NSNikolskiy@sberbank.ru>

phaeton03 committed:
Homework 11 01.11.2021 (#9)

* homework_11
1. Проект на Spring Data

* homework_11
1. Добавлено базовое исключение от которого наследуются все кастомные исключения
2. Снята аннотация Transactional с метода delete в BookService
3. Убрано наследование от CrudRepository

Co-authored-by: u19223669 <NSNikolskiy@sberbank.ru>

phaeton03 committed:
Homework 13 (#10)

* homework_13_MongoDB
1. Приложение на Монго

* homework_13_MongoDB
1. Добавлен README.MD

* homework_13_MongoDB
1. Author сделан как отдельный документ
2. Comments - теперь embedded

Co-authored-by: u19223669 <NSNikolskiy@sberbank.ru>

phaeton03 committed:
Homework 26 spring batch (#13)

* homework_26_SpringBatch
1.spring batch realization

* homework_26_SpringBatch
1. Заменил @Data на @Getter и @Setter в entity

* homework_26_SpringBatch
1. Добавил README.MD

Co-authored-by: u19223669 <NSNikolskiy@sberbank.ru>

phaeton03 committed:
homework_25_Security (#12)

* homework_25_Security
1. Реализована функциональность Аутентификации и Авторизации
2. Добавлены view и MVC контроллеры

* homework_25_Security
1. убрана Transactional с интерфейсов
2. убран EnableWebSecurity

Co-authored-by: u19223669 <NSNikolskiy@sberbank.ru>

phaeton03 committed:
Revert "Homework 26 spring batch (#13)"

This reverts commit a37638d135c058eeca1ab48af8c70880b29e3594.

phaeton03 committed:
Merge pull request #14 from phaeton03/revert-13-homework_26_SpringBatch

Revert "Homework 26 spring batch"

phaeton03 committed:
Homework 26 spring batch (#15)

* homework_26_SpringBatch
1.spring batch realization

* homework_26_SpringBatch
1. Заменил @Data на @Getter и @Setter в entity

* homework_26_SpringBatch
1. Добавил README.MD

phaeton03 committed:
Homework 29 spring integration (#16)

* homework_29_SpringIntegration
1. Реализация жизненного цикла животного через Spring Integration

* homework_29_SpringIntegration
1. Добавление README.MD

phaeton03 committed:
homework_30_Actuator (#17)

1. Добавлен актуатор и кастомные метрики для отслеживания приложение
2. Добавлен CustomHealthIndicator

phaeton03 committed:
homework_32_Actuator (#18)

1. развертка приложения с помощью Миникуба
